tracker-main/main.py

```diff
@@ -37,72 +37,73 @@ def start(update, context):
 #Creates add function apply
 def add(update, context):
     #if there are not enough informations/arguments as update and context return ERORR
-    if len(context.args) < 4:
-        context.bot.send_message(chat_id=update.message.chat_id, text="Please provide a blockchain + Wallet address + Name + address type.\n->Example: /add WalletType 0xadress WalletName AddressType")
+    if len(context.args) < 2:
+        context.bot.send_message(chat_id=update.message.chat_id, text="Please provide a blockchain + Wallet address + Name + address type.\n->Example: /add blockchain 0xadress WalletName (blockchain will be set as ETH if not provided)")
```

Comment:
Please provide a blockchain + Wallet address + Name + address type.\n->Example: /add blockchain 0xadress WalletName (blockchain will be set as ETH if not provided)

to

Please provide blockchain {optional} + wallet address + name.\n->Example: /add blockchain (default: ETH) address name